src/components/ui/color-pickers/ColorPickers.vue

```diff
@@ -37,8 +37,8 @@
             <div class="col-md-1">
               <colored-square :value="simpleColor"/>
             </div>
-            <div class="col-md-4">
-              <vuestic-simple-color-picker v-model="simpleColor"/>
+            <div class="col-md-2">
+              <vuestic-simple-color-picker :palette="['#4ae387', '#49b2e3', '#f81953', '#161616', '#f7cc36']" v-model="simpleColor"/>
```

Comment:
? Would make sense to rename to **palette picker** based on interface. Nice wording BTW.